src/gluonts/evaluation/_base.py

```diff
@@ -30,6 +30,7 @@
 
 import numpy as np
 import pandas as pd
+import toolz
```

Comment:
Personally, I like to import things directly from toolz: `from toolz import valmap, ...`

Makes it easier to read at places where it is used.

Reply:
Alrighty, changed it